### DISCUSSION TITLE
How to create a new directory

### DISCUSSION THREAD

**Discussion by phantomshadow4** [community/community#119654] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi. So... I'm new to using GitHub and I want to ask this; How can I create a directory?

**Comment by markolalovic** [comment, marked as answer]
Hi, you can first try [creating a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories) and then create a directory within it on your computer. Finally, [push the changes to your new repository](https://docs.github.com/en/get-started/using-git/pushing-commits-to-a-remote-repository).
